.github/workflows/job_jax_models_tests.yml

```diff
@@ -0,0 +1,136 @@
+name: JAX/Flax Models tests
+
+on:
+  workflow_call:
+    inputs:
+      runner:
+        description: 'Machine on which the tests would run'
+        type: string
+        required: true
+      container:
+        description: 'JSON to be converted to the value of the "container" configuration for the job'
+        type: string
+        required: false
+        default: '{"image": null}'
+      model_scope:
+        description: 'Scope of models for testing.'
+        type: string
+        required: true
+
+permissions: read-all
+
+jobs:
+  JAX_Models_Tests:
+    name: JAX/Flax Models tests
+    timeout-minutes: ${{ inputs.model_scope == 'precommit' && 35 || 35 }}
+    runs-on: ${{ inputs.runner }}
+    container: ${{ fromJSON(inputs.container) }}
+    defaults:
+      run:
+        shell: bash
+    env:
+      DEBIAN_FRONTEND: noninteractive # to prevent apt-get from waiting user input
+      OPENVINO_REPO: ${{ github.workspace }}/openvino
+      INSTALL_DIR: ${{ github.workspace }}/install
+      INSTALL_TEST_DIR: ${{ github.workspace }}/install/tests
+      MODEL_HUB_TESTS_INSTALL_DIR: ${{ github.workspace }}/install/tests/model_hub_tests
+    steps:
+      - name: Check sudo
+        if: ${{ runner.os == 'Linux' }}
+        run: if [ "$(id -u)" -eq 0 ]; then apt update && apt --assume-yes install sudo; fi
```

Comment:
This also could be removed as you are using Docker with a self-hosted runner: https://github.com/openvinotoolkit/openvino/pull/26542/files#r1755365985.

Reply:
Fixed.